### PR TITLE
feat : Added various text utilities

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Screenshot (338)

### CODE REVIEW

**Comment by shuklahimanshu81** [comment]
Heading text.
Grey text.
Strike text.
Center text.

File: styles.css [hunk 1]
```diff
@@ -33,55 +33,43 @@
...
-      
-  }
-
-  nav{
+}
+nav{
       align-items: center;
       display: flex;
       flex-wrap: wrap;
       height: 125px;
```
Comment (AnuragSahu11) [inline styles.css:55]: Could use rem for this big size

**Review by AnuragSahu11 — COMMENTED** [review]
Looks good

File: styles.css [hunk 2]
```diff
@@ -33,55 +33,43 @@
...
-  }
-  .navigation{
-     
+}
+.navigation{
       display: flex;
       gap: 32px;
       margin-left: auto;
       margin-right: 2rem;
       padding: 8px 8px;
```
Comment (vidhi-thakur) [inline styles.css:62]: Same as above, could use rem

**Review by vidhi-thakur — APPROVED** [review]

**Pull request merged**